main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma",
+    ]
+
+    actors = [
+        ("George", "Klooney"),
```

Comment:
The name 'Klooney' is corrected later in the code to 'Clooney'. Consider using the correct spelling initially to avoid unnecessary updates.